src/main/java/org/tron/common/runtime/vm/program/invoke/ProgramInvokeImpl.java

```diff
@@ -323,18 +310,13 @@ public boolean equals(Object o) {
     if (deposit != null ? !deposit.equals(that.deposit) : that.deposit != null) {
       return false;
     }
-    //if (storage != null ? !storage.equals(that.storage) : that.storage != null) return false;
-    if (timestamp != null ? !timestamp.equals(that.timestamp) : that.timestamp != null) {
-      return false;
-    }
-
-    return true;
+    return timestamp != null ? timestamp.equals(that.timestamp) : that.timestamp == null;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Convert this usage of the ternary operator to an "if"/"else" structure. ([squid:S1774](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1774))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1681&issues=AWbnEHR-SgN4uCcljsr4&open=AWbnEHR-SgN4uCcljsr4)

[ISSUE_KEY]: AWbnEHR-SgN4uCcljsr4